### PR TITLE
Fixed tear down null pointer exception

### PR DESCRIPTION
**Pull request by lchrusciel**

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
@lchrusciel what can cause the `client` to be `null`?

**Comment by lchrusciel** [comment]
Wrong phpunit.xml configuration, problems with services and so on... Everything that will cause problems with creating krenel. Without this line you will see only null pointer exception. Otherwise its usually shows reason, why it is not working.

**Comment by pjedrzejewski** [comment]
I am not sure about this change. :/

**Comment by lchrusciel** [comment]
Trust me, it is really usefull when it comes to debugging

**Comment by michalmarcinkowski** [comment]
I'm for avoiding `null` pointer exceptions everywhere we can :+1:

File: src/ApiTestCase.php [hunk 1]
```diff
@@ -34,8 +34,10 @@ public function setUp()
 
     public function tearDown()
     {
-        foreach ($this->client->getContainer()->getMockedServices() as $id => $service) {
-            $this->client->getContainer()->unmock($id);
+        if (null !== $this->client){
```
Comment (pjedrzejewski) [inline src/ApiTestCase.php]: CS

**Comment by pjedrzejewski** [comment]
Thanks Łukasz!

**Pull request merged**